components/MediaCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <section>
+    <h3 class="py-12 capitalize">Latest {{ mediaType }}</h3>
+    <article
+      class="items-center grid grid-cols-1 pb-12 lg:gap-12 lg:grid-cols-9"
+    >
+      <div class="lg:col-span-5">
+        <h4 class="text-xl lg:mb-4">
+          {{ details.title }}
+        </h4>
+        <p>
+          {{ formatDateWithDots(details.date || details.pubDate) }}
+        </p>
+        <p>
+          {{ details.description }}
+        </p>
+      </div>
+      <nuxt-img
+        class="order-first h-auto m-auto max-w-md w-full pb-10 lg:col-span-2"
+        format="webp"
+        height="500"
```

Comment:
I'm not sure if images would always be `500x500`. Perhaps these should be props? Maybe one `img` prop that is an object with src and dimensions?

Reply:
Trying updating but not sure why this syntax won't work?
<img width="617" alt="Screen Shot 2022-03-07 at 3 42 01 PM" src="https://user-images.githubusercontent.com/62962173/157136981-4286e449-6e61-4913-b3aa-e1bdb34c0335.png">

<img width="704" alt="Screen Shot 2022-03-07 at 3 42 40 PM" src="https://user-images.githubusercontent.com/62962173/157137026-1396bfd0-448b-4b4b-b663-5d0c2cc01c91.png">

Reply:
I think we could use a `default` on the prop to set them to `500` if they were not passed in. Not sure why nullish coalescing is not working here.